### PR TITLE
Update links to Katacoda tutorials to point to official versions

### PR DESCRIPTION
**Pull request by AlanGreene**
# Changes

Update links to the Katacoda tutorials to point to the official versions
at https://katacoda.com/tektoncd and update the dashboard scenario naming
to be consistent with the others (i.e. drop the redundant `tekton-` from the
name + URL).

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/website/blob/master/CONTRIBUTING.md)
for more details._

### CODE REVIEW

**Comment by afrittoli** [comment]
Thanks. I checked the preview and I get a 404 on https://katacoda.com/tektoncd/scenarios/dashboard ?

**Comment by AlanGreene** [comment]
This PR renames the folder for the Dashboard tutorial so once merged Katacoda will rebuild and it will be at https://katacoda.com/tektoncd/scenarios/dashboard instead of `tekton-dashboard`

**Comment by afrittoli** [comment]
> This PR renames the folder for the Dashboard tutorial so once merged Katacoda will rebuild and it will be at https://katacoda.com/tektoncd/scenarios/dashboard instead of `tekton-dashboard`
> 
> I've pushed this PR branch to my repo's main branch to verify on Katacoda: https://katacoda.com/alangreene-tektoncd/scenarios/dashboard

Oh, I see, thanks!
/lgtm

File: layouts/try/baseof.html [hunk 1]
```diff
@@ -48,7 +48,7 @@ <h2>Interactive Tutorials</h1>
             <div class="col-lg-8 mr-2 ml-2">
                 <div class="row">
                     <div class="col-md-4 pl-1 pr-1 pb-3">
-                        <a href="https://katacoda.com/ncskier/scenarios/tekton-dashboard" target="_blank" rel="noopener">
+                        <a href="https://katacoda.com/tektoncd/scenarios/dashboard" target="_blank" rel="noopener">
```
Comment (popcor255) [inline layouts/try/baseof.html:51]:
```suggestion
                        <a href="https://katacoda.com/tektoncd/scenarios/tekton-dashboard" target="_blank" rel="noopener">
```
nit: the url is incorrect.

**Review by popcor255 — CHANGES_REQUESTED** [review]

File: layouts/try/baseof.html (hunk 1 above)
Comment (AlanGreene) [reply layouts/try/baseof.html:51]: See my comment above in response to Andrea's question, this is an intentional change.

**Comment by popcor255** [comment]
oh, sorry.
/lgtm

**Review by afrittoli — APPROVED** [review]
/approve

**Pull request merged**